data/categories.txt

```diff
@@ -4681,6 +4704,8 @@ fa:مزرعه
 # Skipped if empty name
 waterway-river|waterway-stream|waterway-riverbank
 en:River
+en-GB:River|stream
```

Comment:
For these cases (when the main word is the same in dialects, but search synonyms may be different), it's better to leave only en: and add all additional synonyms there (may be with a number hint).

Reply:
Sure.